boxoffice/views/admin_discount.py

```diff
@@ -152,13 +154,40 @@ def admin_edit_discount_policy(discount_policy):
             errors=discount_policy_form.errors)
 
 
+def jsonify_delete_discount_policy(discount_policy_dict):
```

Comment:
This function is view logic. It is not a renderer. It should be a part of the view function.

Reply:
Will renaming the function to `delete_discount_policy` work? It's been extracted out to make it work with `render_with`.

Reply:
`render_with` is the wrong tool here. Flask will let you specify separate handlers by content type, so use that and make separate views.